lightning/src/offers/invoice_request.rs

```diff
@@ -989,7 +989,14 @@ impl VerifiedInvoiceRequest {
 		InvoiceWithDerivedSigningPubkeyBuilder
 	);
 
-	pub(crate) fn fields(&self) -> InvoiceRequestFields {
+	/// Fetch the [`InvoiceRequestFields`] for this verified invoice.
+	///
+	/// These are fields which we expect to be useful when receiving a payment for this invoice
+	/// request, and include the returned [`InvoiceRequestFields`] in the
+	/// [`PaymentContext::Bolt12Offer`].
+	///
+	/// [`PaymentContext::Bolt12Offer`]: crate::blinded_path::payment::PaymentContext::Bolt12Offer
+	pub fn fields(&self) -> InvoiceRequestFields {
```

Comment:
Didn't you want to do the fuzzy_ submodule thing here to expose it to the fuzzer?